### DISCUSSION TITLE
pInputText in p-table

### DISCUSSION THREAD

**Discussion by adellaposta** [primefaces/community#342] [PrimeNG]
I have a p-table with a caption template. Inside the caption template i have an input to filter globally inside the results of the table. 
If i use the input like this: 
`<input pInputText type="text" placeholder="Search Keyword" size="50" class="p-inputtext-sm" />`
in console i get the error "object unsubscribed", but if i use the input like this:
`<input type="text" placeholder="Search Keyword" size="50" class="p-inputtext p-component p-element p-inputtext-sm" />`
i don't have any error.
Is this a standard behaviour or am i missing something?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
